<launch>
    <param name="robot_description" command="cat '$(find autonomus_transport_industrial_system)/urdf/robot1.urdf'"/>

    <!-- send fake joint values -->
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="False"/>
    </node>

    <!-- Combine joint values -->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find autonomus_transport_industrial_system)/worlds/obstacle_map.world"/>
        <arg name="paused" value="false"/>
        <arg name="use_sim_time" value="true"/>
        <arg name="gui" value="true"/>
        <arg name="headless" value="false"/>
        <arg name="debug" value="false"/>
        <remap from="tf" to="gazebo_tf"/>
    </include>

    <node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf" args="-urdf -model my_robot -x -6 -y 0 -z 0.1  -param robot_description" />

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find autonomus_transport_industrial_system)/rviz/rviz_rm.rviz"/>

    <node pkg="tf" type="static_transform_publisher" name="camera_front" args="0.0 0.0 0.0 -1.57 0 -1.57 camera_front_link camera_front 100" />
    <node pkg="tf" type="static_transform_publisher" name="camera_right" args="0.0 0.0 0.0 -1.57 0 -1.57 camera_right_link camera_right 100" />
    <node pkg="tf" type="static_transform_publisher" name="camera_left" args="0.0 0.0 0.0 -1.57 0 -1.57 camera_left_link camera_left 100" />

</launch>